website/project/model.py

```diff
@@ -2119,7 +2119,11 @@ def fork_node(self, auth, title='Fork of '):
                 if forked_node is not None:
                     forked.nodes.append(forked_node)
 
-        forked.title = title + forked.title
+        if title == 'Fork of ' or title == '':
```

Comment:
In order to avoid special-casing `"Fork of"` how about making `title` default to `None`?

```python
def fork_node(..., title=None):
    PREFIX = 'Fork of '
    if title is None:
         title = PREFIX + forked.title
```